lib/utils/getSchemeFromUrl.cjs

```diff
@@ -1,15 +1,14 @@
 'use strict';
 
-const { URL } = require('url');
-
```

Comment:
[note] `URL` is now a global object. See https://nodejs.org/api/url.html#class-url